horace_core/sqw/@rundatah/private/calc_sqw_.m

```diff
@@ -44,6 +43,28 @@
 if hor_log_level>-1
     disp('Calculating projections...');
 end
+proj = instr_proj(obj.lattice,obj.efix,'emode',obj.emode);
+%%
+ax_block = proj.get_proj_axes_block(pix_db_range_in,grid_size_in);
+
+    if isempty(qspec)
+        qspec_provided = false;
+        if isempty(detdcn)
+            detdcn = calc_detdcn(obj.det_par);
+        end
+        [qspec,en]=obj.calc_qspec(detdcn);
+        ucoords = [spec_to_cc*qspec;en];
+    else
+        ucoords = [spec_to_cc*qspec(1:3,:);qspec(4,:)];
+        qspec_provided = true;
+    end
```

Comment:
indentation for this block is not consistent

Reply:
fixed